R/misc.R

```diff
@@ -659,6 +659,24 @@ check_training_set <- function(x, rec, fresh) {
   x
 }
 
+check_newdata_columns <- function(object, new_data) {
```

Comment:
Can we call this function `check_new_data_columns()`?

Reply:
Done!